mantidproject/mantid
# Remove ILL indirect deprecated code

This PR removes the old version of the loader, 2 deprecated algorithms, and one GUI tab for indirect ILL.

**To test:**
1. Set at ISIS, go to `Interfaces>Indirect>Data Reduction`
2. You should see 7 tabs, from which first 3 specific to ISIS
3. Set at ILL, go to `Interfaces>Indirect>Data Reduction`
4. Now you should see 5 tabs, only first of which specific to ILL (Energy Transfer).
5. [Optional] If you are familiar with Indirect, check if there is something else that needs to be removed and I missed?



Fixes #20083 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
